site/content/en/docs/manual/advanced/ai-tools.md

```diff
@@ -1,155 +1,309 @@
+| --------------------------------------------------------- | --------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | ------------------------------ |
+| Deep extreme <br>cut (DEXTR)                              | This is an optimized version of the original model, <br>introduced at the end of 2017. It uses the <br>information about extreme points of an object <br>to get its mask. The mask is then converted to a polygon. <br>For now this is the fastest interactor on the CPU.                                                         | ![](/images/dextr_example.gif) |
+| Feature backpropagating <br>refinement <br>scheme (f-BRS) | The model allows to get a mask for an <br>object using positive points (should be <br>left-clicked on the foreground), <br>and negative points (should be right-clicked <br>on the background, if necessary). <br>It is recommended to run the model on GPU, <br>if possible.                                                     | ![](/images/fbrs_example.gif)  |
+| High Resolution <br>Net (HRNet)                           | The model allows to get a mask for <br>an object using positive points (should <br>be left-clicked on the foreground), <br> and negative points(should be <br>right-clicked on the background, <br> if necessary). <br>It is recommended to run the model on GPU, <br>if possible.                                                | ![](/images/hrnet_example.gif) |
+| Inside-Outside-Guidance<br>(IOG)                          | The model uses a bounding box and <br>inside/outside points to create a mask. <br>First of all, you need to create a bounding<br> box, wrapping the object. <br>Then you need to use positive <br>and negative points to say the <br>model where is <br>a foreground, and where is a background.<br>Negative points are optional. | ![](/images/iog_example.gif)   |
 
-![](/images/iog_example.gif)
+<!--lint enable maximum-line-length-->
 
-## Detectors
+### Detectors
 
-Detectors are used to automatically annotate one frame. Supported DL models are suitable only for certain labels.
+Use detectors to automatically annotate one frame.
 
-- Before you start, click the `magic wand` on the controls sidebar and select the `Detectors` tab.
-  You need to match the labels of the DL model (left column) with the labels in your task (right column).
-  Then click `Annotate`.
+Supported DL models are suitable only for certain labels.
```

Comment:
Each model is trained on a dataset. In many cases it is COCO, Pascal VOC. The model supports labels that were available in the training dataset. Need to clarify that the list of labels can be viewed on a model page for each such model. Just an idea.